src/redux/types/index.js

```diff
@@ -1,3 +1,13 @@
+export const IS_LOADING = 'IS_LOADING'
+
+// Image daily
 export const REQUEST_IMAGES = 'REQUEST_IMAGES'
 export const FETCH_IMAGES_SUCCESS = 'FETCH_IMAGES_SUCCESS'
-export const IS_LOADING = 'IS_LOADING'
+
+// Images Gallery
+export const REQUEST_IMAGES_GALLERY = 'REQUEST_IMAGES_GALLERY'
+export const FETCH_IMAGES_GALLERY_SUCCESS = 'FETCH_IMAGES_GALLERY_SUCCESS'
+export const SET_SEARCH_WORD = 'SET_SEARCH_WORD'
+export const SET_CURRENT_PAGE = 'SET_CURRENT_PAGE'
+export const SET_CURRENT_YEARS_DATE = 'SET_CURRENT_YEARS_DATE'
+export const SET_CURRENT_YEARS_AFTER = 'SET_CURRENT_YEARS_AFTER'
```

Comment:
it's time to think about Grouping these actions and separate them into different files. That file will grow up a lot and it will become not possible to read or find action here. we need to split them, let's discuss tomorrow